client/reader/stream/index.jsx

```diff
@@ -211,7 +211,10 @@ class ReaderStream extends React.Component {
 		}
 
 		if ( config.isEnabled( 'reader/combined-cards' ) ) {
-			items = combineCards( items );
+			// Create combined cards unless we're on a site or feed stream
+			if ( ! startsWith( store.id, 'feed:' ) && ! startsWith( store.id, 'site:' ) ) {
```

Comment:
I thought about that - but what's the best format to do that (capturing the `startsWith` malarkey too)? A regex?